### PR TITLE
Add contributor set-up instructions

### PR DESCRIPTION
**Pull request by samwilson**
Change the main install instructions to target developers, because they're more likely to be the ones following this. #wmhack

### CODE REVIEW

**Comment by waldyrious** [comment]
As previously promised, I've proposed changes to this branch in #475.

File: README.md [hunk 1]
```diff
@@ -19,56 +18,116 @@ Requirements
...
 ## CLI Usage
 
 ### app:check
 
-Run epubcheck on books. With no options set, this will check 10 random books from English Wikisource. Note that the random 10 will be cached (for repeatability) unless you use <info>--nocache</info>.
+Run epubcheck on books. With no options set, this will check 10 random books from English Wikisource.
+Note that the random 10 will be cached (for repeatability) unless you use <info>--nocache</info>.
 
-    app:check [-l|--lang LANG] [--nocache] [-t|--title TITLE] [-c|--count COUNT] [-s|--namespaces NAMESPACES]
+```console
```
Comment (samwilson) [inline README.md:128]: @waldyrious I didn't notice this before, sorry. This section of CLI usage is auto-generated (by [this thing](https://packagist.org/packages/samwilson/console-readme-generator)), so shouldn't be changed by hand. I thought we had a CI check in place to guard against this; I'm adding such a thing in #476.

Comment (waldyrious) [reply README.md:128]: Sure, no worries. Would you like me to revert the changes to the "CLI Usage" section, and keep the changes to the other sections?

Comment (waldyrious) [reply README.md:128]: In the meantime, I submitted https://github.com/samwilson/console-readme-generator/pull/7 to change this formatting in the source.

Comment (samwilson) [reply README.md:128]: Thanks! And no, it's fine to leave them as-is here, I'll merge that change and then we can update the package here.

**Comment by samwilson** [comment]
Okay, I think this is good to go now.

File: README.md [hunk 2]
```diff
@@ -19,56 +18,115 @@ Requirements
   * `fonts-gubbi`
   * `fonts-opendyslexic`
 
-Installation
-============
+## Development installation (without Docker)
+
+This section deals with a full installation of WS Export on the local development machine. For Docker installation instructions, see below.
```
Comment (waldyrious) [inline README.md:23]:
```suggestion
This section deals with a full installation of WS Export on the local development machine. For Docker installation instructions, see [below](#docker-developer-environment).
```

File: README.md [hunk 3]
```diff
@@ -19,56 +18,115 @@ Requirements
...
-  * Bind address for docker enviroments
-```bash
-$ ./bin/console toolforge:ssh --bind-address=0.0.0.0
-```
+   ```console
+   symfony server:start -d
+   ```
+
+4. Open your web browser to e.g. <http://localhost:8000>
+   and the basic operations should be working with the following requirements:
```
Comment (waldyrious) [inline README.md:62]: I think "limitations" may be a better word choice than "requirements". WDYT?

File: README.md [hunk 4]
```diff
@@ -19,56 +18,115 @@ Requirements
...
+
+5. Install optional dependencies:
+
+   **Calibre:**
+   In order to export to PDF, plain text, RTF, or Mobi formats
+   you must install [Calibre](https://calibre-ebook.com)
+   so that the tool can use the `ebook-convert` command.
+
+   **epubcheck:**
+   To validate exported ebooks (with the `./bin/console app:check` command),
```
Comment (waldyrious) [inline README.md:77]:
```suggestion
   To validate exported ebooks (with the [`./bin/console app:check` command](#appcheck)),
```

By the way, if the `./bin/console` is needed to run the command, why isn't it included in the "CLI Usage" section below? :thinking:

File: README.md [hunk 5]
```diff
@@ -19,56 +18,115 @@ Requirements
...
+   you must install [epubcheck](https://github.com/w3c/epubcheck).
+   If it's not installed at `/usr/bin/epubcheck` then
+   set the `EPUBCHECK_JAR` environment variable.
```
Comment (waldyrious) [inline README.md:80]:
```suggestion
   If it installed at a location other than `/usr/bin/epubcheck`, then
   set the `EPUBCHECK_JAR` environment variable to the correct path.
```

File: README.md [hunk 6]
```diff
@@ -19,56 +18,115 @@ Requirements
...
+   the `fonts-linuxlibertine` package.
+   You can also install any other fonts you want to use for exporting books.
+
+6. WS Export uses two database connections:
+   firstly, to its own database to store download statistics;
+   and secondly, it connects to the Wikimedia database replicas
+   to fetch information about Wikisource contributors
+   for the credits list that can be included at the end of exported books.
+
+   For the statistics recording database you create the the database
```
Comment (waldyrious) [inline README.md:93]:
```suggestion
   For the statistics recording database you need to create the the database
```

File: README.md [hunk 7]
```diff
@@ -19,56 +18,115 @@ Requirements
...
+
+   ```
+   ./bin/console doctrine:migrations:migrate`
+   ```
+
+8. To connect to the Wikimedia replicas, this tool uses
+   the [Toolforge Bundle](https://github.com/wikimedia/ToolforgeBundle),
+   and connects to [multiple databases](https://github.com/wikimedia/ToolforgeBundle#replicas-connection-manager).
+
+   Set replicas credentials in your `.env.local` file according to the Bundle's documentation.
```
Comment (waldyrious) [inline README.md:110]:
```suggestion
   Set the replicas' credentials in your `.env.local` file
   according to the Bundle's documentation.
```

File: README.md [hunk 8]
```diff
@@ -19,56 +18,115 @@ Requirements
...
+
+9. Establish an SSH tunnel to the replicas:
+
+  ```console
+  ./bin/console toolforge:ssh
+  ```
+
+10. At this point, you should be able to use all of the functionality
+    of WS Export both via the web interface and CLI.
```
Comment (waldyrious) [inline README.md:119]:
```suggestion
    of WS Export, via both the web interface and the CLI.
```

File: README.md (hunk 3 above)
Comment (samwilson) [reply README.md]: Done!

**Pull request merged**